resources/lang/en/labels.php

```diff
@@ -4,4 +4,12 @@
     'label_promotion_admin' => 'Promotion Admin',
     'label_logout'   => 'Log Out',
     'label_dasboard' => 'Dasboard',
+    'label_city'     => 'City',
```

Comment:
this file name is `labels.php`, so you don't need to add `label_` prefix to every key name